Examples/Sources/ChatbotExample/ChatbotApp.swift

```diff
@@ -0,0 +1,47 @@
+import DefaultBackend
+import SwiftCrossUI
+
+#if canImport(SwiftBundlerRuntime)
+    import SwiftBundlerRuntime
+#endif
+
+// MARK: - Main App
+
+@main
+@HotReloadable
+struct ChatbotApp: App {
+    @SwiftCrossUI.State private var viewModel = ChatbotViewModel()
+    
+    var body: some Scene {
+        WindowGroup("ChatBot") {
+            #hotReloadable {
+                NavigationSplitView {
+                    // Sidebar content
```

Comment:
Remove these section comments from this app body. The parts are all relatively self explanatory already imo.